addOns/network/src/test/java/org/zaproxy/addon/network/GlobalExclusionsOptionsUnitTest.java

```diff
@@ -215,6 +219,71 @@ void shouldMigrateCoreOptions() {
         assertExclusion(0, "Name", "Value", false);
     }
 
+    @Test
+    void shouldHaveAllDefaultExclusionsCaseInsensitive() {
+        // Given / When
+        List<GlobalExclusion> exclusions = options.getGlobalExclusions();
+        // Then
+        Map<String, String> patterns = new HashMap<>();
+        exclusions.forEach(ex -> patterns.put(ex.getName(), ex.getValue()));
+        List<String> errors = new ArrayList<>();
+        patterns.entrySet()
+                .forEach(
+                        entry -> {
+                            if (!entry.getValue().startsWith("(?i)")) {
+                                errors.add(entry.getKey());
+                            }
+                        });
+        assertThat(errors, is(empty()));
+    }
+
+    @Test
+    void shouldInsertCaseInsensitivityOnDefaultPatternsOnUpdateFromVersion1() {
```

Comment:
Why just version 1? Aren't the older values also missing the flag?

Reply:
I did change this to ....ToVersio2

Reply:
The test should also be changed to test the update without version.

Reply:
Will tackle that later today (Monday).

Reply:
```xml
<config>
    <globalexcludeurl>
        <url_list>
            <url>
                <regex>^.*\.(?:gif|jpe?g|png|ico|icns|bmp)$</regex>
                <description>Extension - Image (ends with .extension)</description>
                <enabled>true</enabled>
            </url>
            <url>
                <regex>^.*\.(?:mp[34]|mpe?g|m4[ap]|aac|avi|mov|wmv|og[gav])$</regex>
                <description>Extension - Audio/Video (ends with .extension)</description>
                <enabled>true</enabled>
            </url>
        </url_list>
        <confirmRemoveToken>true</confirmRemoveToken>
    </globalexcludeurl>
</config>
```

Reply:
I didn't get to this as planned, will tackle it early Tuesday.